testutil/integration/evmos/network/network.go

```diff
@@ -249,6 +249,11 @@ func (n *IntegrationNetwork) GetDenom() string {
 	return n.cfg.denom
 }
 
+// GetOtherDenoms returns network's other supported denoms
+func (n *IntegrationNetwork) GetOtherDenoms() []string {
```

Comment:
Might be a stupid question.. Is this the way to query this information? or should it be a bank grpc query to a respective account and see wehat type of denoms it owns?